structured_experiments/2022.07.19.notebook_template/experiment.MD

```diff
@@ -0,0 +1,46 @@
+<!--DESC Template folder to use for experiments-->
+ Hypothesis
+TODO: Create Template
```

Comment:
Just fill in the TODOs here and your all set 👍

Reply:
Got it done!